src/common/types/types.cpp

```diff
@@ -36,6 +37,8 @@ std::string PhysicalTypeUtils::physicalTypeToString(PhysicalTypeID physicalType)
         return "UINT16";
     case PhysicalTypeID::UINT8:
         return "UINT8";
+    case PhysicalTypeID::INT128:
+        return "INT128";
```

Comment:
Guess, we should ignore the coverage check for the `PhysicalTypeUtils::physicalTypeToString` function.